.github/workflows/_create_prepare_aks_cluster.yml

```diff
@@ -0,0 +1,116 @@
+name: Create and Prepare AKS Cluster
+
+on:
+  workflow_call:
+    outputs:
+      cluster-name:
+        description: AKS Cluster Name With Randomly Generated Suffix
+        value: ${{ jobs.create-prepare-aks-cluster.outputs.cluster-name }}
+  workflow_dispatch:
```

Comment:
Since we won't be manually calling this workflow, you can remove the workflow_dispatch.